content/zh/docs/concepts/extend-kubernetes/api-extension/custom_resources.md

```diff
@@ -0,0 +1,524 @@
+-->
+|一旦CRD创建，无需后续支持。任何错误修复都是正常Kubernetes Master升级的一部分。|可能需要定期从上游拾取错误修复，并重建和更新API集合服务器。|
+<!--
+| No need to handle multiple versions of your API. For example: when you control the client for this resource, you can upgrade it in sync with the API. | You need to handle multiple versions of your API, for example: when developing an extension to share with the world. |
+-->
+|无需处理API的多个版本。例如：当您控制此资源的客户端时，可以将其与 API 同步升级。|你需要处理API的多个版本，例如：当开发一个扩展，与世界分享时。|
+<!--
+### Advanced features and flexibility
+-->
+高级功能和灵活性
+<!--
+Aggregated APIs offer more advanced API features and customization of other features, for example: the storage layer.
+-->
+API集合提供更高级的API特性以及其他功能的自定义，例如：存储层。
+<!--
+| Feature | Description | CRDs | Aggregated API |
+-->
+|特性|描述|CRDs|API集合|
+			
+<!--
+| Validation | Help users prevent errors and allow you to evolve your API independently of your clients. These features are most useful when there are many clients who can't all update at the same time. | Yes.  Most validation can be specified in the CRD using [OpenAPI v3.0 validation](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation).  Any other validations supported by addition of a [Validating Webhook](/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9). | Yes, arbitrary validation checks |
+-->
+|验证|帮助用户避免错误并且允许您独立于客户端发展API。|是的。大多数验证可以在CRD中使用OpenAPI v3.0来验证。其他验证可以通过添加验证的网络挂接（webhook）来支持。|是的，任意验证检查。|
+<!--
+| Defaulting | See above | Yes, via a [Mutating Webhook](/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9); Planned, via CRD OpenAPI schema. | Yes |
+|违约|见上文|是的，通过突变的网络挂接；规划，通过CRD OpenAPI架构。|是的|
+-->
+<!--
+| Multi-versioning | Allows serving the same object through two API versions. Can help ease API changes like renaming fields. Less important if you control your client versions. | [Yes](/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning) | Yes |
+-->
+|多版本|允许通过两个API版本为同一对象提供服务。可帮助简化 API 更改,如重命名字段。如果您控制客户端版本,则不太重要。|是的|是的|
+<!--
+| Custom Storage | If you need storage with a different performance mode (for example, time-series database instead of key-value store) or isolation for security (for example, encryption secrets or different | No | Yes |
+-->
+|自定义存储|如果需要不同性能模式的存储（例如，时间序列数据库而不是密钥值存储）或为了安全进行隔离（例如，加密机密或不同的）|不是|是的|
+| Custom Business Logic | Perform arbitrary checks or actions when creating, reading, updating or deleting an object | Yes, using [Webhooks](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks). | Yes |
+<!--
+| Scale Subresource | Allows systems like HorizontalPodAutoscaler and PodDisruptionBudget interact with your new resource | [Yes](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#scale-subresource)  | Yes |
+-->
+|缩放子资源|允许水平Pod自动缩放器和Pod中断预算等系统与您的新资源进行交互|是的|是的|
```

Comment:
I'd expect this line to include the original terms: “Scale”, “HorizontalPodAutoscaler” and “PodDisruptionBudget”.